archivist_samples/sbom_document/software_package.py

```diff
@@ -0,0 +1,434 @@
+# pylint:disable=missing-function-docstring      # docstrings
```

Comment:
Is this an exact copy of software_package.py from the other sample, or are there some particular changes you'd like reviewed?

Reply:
This is an exact copy, with the addition of using the `OnboardingSampleID` as the identifier for if the asset exists or not.